### ISSUE TITLE
Weird gray stripes when displaying content elements in the backend

### ISSUE THREAD

**Issue by agoat** [contao/core-bundle#1426]
Some content elements in version 4.5 now have a grey stripe (added with 29d06595ee11b59cfbfb8c97bfc8a7fbece4e4bb).
Did this happen on purpose?

![grey_bars](https://user-images.githubusercontent.com/1402434/37332619-785af020-26a7-11e8-82a9-93b5f84ac901.jpg)

IMO it's a bit confusing if download has a white background while downloads shown with a gray background.

The responsible css statement has been changed as follows:
```diff
-.tl_content.wrapper_start,.tl_content.wrapper_separator,.tl_content.wrapper_stop {
+.parent_view ul {
 	background:#f3f3f5;
 }
```
All `ul` elements in the backend that are displayed in sorting mode 4 are now highlighted in gray.

I've come across it because an extension now also shows a gray stripe.

**Comment by leofeyer** [comment]
Does this affect Contao 4 as well?

**Comment by agoat** [comment]
> Does this affect Contao 4.4 as well?

No. The `.parent_view ul` style declaration only exists in 4.5+.

**Comment by leofeyer** [comment]
Fixed in 4327a405669be059dd309645fe53c7cff70eadf4.
